### ISSUE TITLE
Option to restore deleted instances

### ISSUE THREAD

**Issue by joedolson** [joedolson/my-calendar#20]
Now that I have a record of deleted instances, should look at an option to restore them.

**Comment by joedolson** [comment]
No, this is about restoring single dates that were deleted from a recurring event. E.g., somebody uses the Add Occurrence UI to add a couple additional dates, later deletes one, then wants to restore it.

Not really a high priority, I think.

**Comment by joedolson** [comment]
UI possibility: could add a tab to the list of scheduled dates that holds removed dates, and the possibility of restoration from there. Not super complicated, but not a top priority.

**Comment by paaljoachim** [comment]
That sounds interesting! It reuses a known UI that users will recognize.

**Comment by joedolson** [comment]
This is interesting, but not frequently requested and doesn't fit well with the other goals for 3.5. Moving to future release.
